packages/agoric-cli/scripts/get-sdk-package-names.js

```diff
@@ -0,0 +1,33 @@
+#! /usr/bin/env node
+/* global process, Buffer */
+import { spawn } from 'child_process';
+import { basename } from 'path';
+
+const ps = spawn('yarn', ['workspaces', '--silent', 'info'], {
+  stdio: ['ignore', 'pipe', 'inherit'],
+});
+
+// Get Buffers of output.
+const chunks = [];
+ps.stdout.on('data', data => chunks.push(data));
+
+// Wait for the process to exit.
+ps.on('close', code => {
+  if (code !== 0) {
+    throw Error(`yarn info exited with code ${code}`);
+  }
+
+  // Get the output.
+  const json = Buffer.concat(chunks).toString('utf8');
+  // process.stderr.write(json);
+
+  // Write the module.
+  const workspaces = Object.keys(JSON.parse(json)).sort();
+  process.stdout.write(`\
+// DO NOT EDIT - automatically generated by ${basename(
+    new URL(import.meta.url).pathname,
+  )}
+/* eslint-disable comma-dangle,quotes,prettier/prettier */
```

Comment:
I think it should be explicitly excluded from prettier itself, not just from the eslint prettier rule, orelse editors which integrate prettier will still attempt to format the file.
I believe an inline comment will do here (no need to muck with the `.prettierignore` file)

```suggestion
/* eslint-disable comma-dangle,quotes */
// prettier-ignore
```

Reply:
Excellent!  Thank you.